service/server.py

```diff
@@ -332,3 +341,12 @@ def _search_results_page(results, search_term):
 
 def _cookies_page():
     return render_template('cookies.html', username=current_user.get_id())
+
+
+def _create_string_date(datetoconvert, needs_time):
+    # converts to example : 12 August 2014
+    date = datetoconvert.strftime('%d %B %Y')
+    if needs_time:
```

Comment:
How about making a separate method for formatting dates with time? What do you think?